### PR TITLE
Added a few things

### PR DESCRIPTION
**Pull request by Kiwii12**
+actions
+nouns
+adjectives
+eastereggs

### CODE REVIEW

File: res/adjectives.txt [hunk 1]
```diff
@@ -41,8 +41,10 @@ non-standard
 cached
 stored
 formatted
-assembling
-disassembling
 distributed
 virtual
 augmented
+binary
+classical
```
Comment (StrangePan) [inline res/adjectives.txt:48]: Questionable compatibility; feels out-of-place and not commonly used in tech jargon

File: res/adjectives.txt [hunk 2]
```diff
@@ -41,8 +41,10 @@ non-standard
...
+literary
```
Comment (StrangePan) [inline res/adjectives.txt:49]: Questionable compatibility; feels out-of-place and not commonly used in tech jargon

File: res/adjectives.txt [hunk 3]
```diff
@@ -41,8 +41,10 @@ non-standard
...
+multiple
```
Comment (StrangePan) [inline res/adjectives.txt:50]: Questionable compatibility; may not mesh well with several nouns. (e.g. "compiling multiple cache", "turbo-encabulating mutliple Java")

File: res/eastereggs.txt [hunk 4]
```diff
@@ -9,3 +9,7 @@ calculating the answer to life, the universe, and everything
 coping with meaningless existance
 balancing hearthstone classes
 praying to RNGesus
+writing Dan language
+setting multiple fork-bombs
+hoping for tokens at the back of MTG booster packs
+rolling dice for new D&D character
```
Comment (StrangePan) [inline res/eastereggs.txt:15]: Improvement suggestion; "rolling stats for new D&D character"

File: res/eastereggs.txt [hunk 5]
```diff
@@ -9,3 +9,7 @@ calculating the answer to life, the universe, and everything
 coping with meaningless existance
 balancing hearthstone classes
 praying to RNGesus
+writing Dan language
```
Comment (StrangePan) [inline res/eastereggs.txt:12]: Rejected entry; a bit too localized of an inside joke to be included

File: res/nouns.txt [hunk 6]
```diff
@@ -69,3 +69,7 @@ user directory
 cache
 arduino cluster
 neural network
+seg-fault
```
Comment (StrangePan) [inline res/nouns.txt:72]: Conflicting entry; another noun exists with extremely similar meaning
- "seg-faults"

File: res/nouns.txt [hunk 7]
```diff
@@ -69,3 +69,7 @@ user directory
...
+seg-faults
```
Comment (StrangePan) [inline res/nouns.txt:73]: Conflicting entry; another noun exists with extremely similar meaning (I prefer this one over the other)
- "seg-fault"

**Comment by StrangePan** [comment]
@Kiwii12 Pull request reviewed. Please review comments and address issues.

**Comment by StrangePan** [comment]
@Kiwii12 Would you like to make these changes, or shall I close this pull request?

**Comment by Kiwii12** [comment]
Close it. That was a long time ago haha. Tell me if you ever get it windows
compatible:)

**Pull request closed without merging**